mission_control/navigator_missions/nav_missions/symbol_mission.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import txros
+import rospy
+import std_srvs.srv
+from navigator_msgs.msg import DockShape
+import numpy
+from geometry_msgs.msg import Point
+import sensor_msgs.msg
+import nav_msgs.msg
+
+import cv2
+
+from twisted.internet import defer
+
+
+
+error_threshold = 0.01
+move_per_iteration = 1.0;
+#~ square = numpy.array([Point(-1,-1,0), Point(-1,1,0), Point(1,1,0), Point(1,-1,0)])
+#~ square = numpy.array([-1,-1,0,-1,1,0,1,1,0,1,-1,0])
+square = numpy.array([[0, 0, 0],[ 1, 0, 0], [0, 1, 0],[ 1, 1, 0]]).astype(numpy.float64)
+#~ K = numpy.array(numpy.mat("1 0 0;0 1 0;0 0 1"))
+K = numpy.array([[1,0,0],[0,1,0],[0,0,1]]).astype(numpy.float64)
+
+
+def boundingRect(points):
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/) (the python style guides, and the rules we use for all the python on the boat), function names are supposed to be all lowercase separated by underscores (`bounding_rect(points)`). On the sub we were pretty strict about pep8, and its really helpful for people who have to look at your code to have everything (class, variables, and method names among other things) standardized with the rest of the repo.